exec/java-exec/src/test/java/org/apache/drill/exec/store/TestImplicitFileColumns.java

```diff
@@ -110,4 +111,20 @@ public void testImplicitColumnsForParquet() throws Exception {
         .go();
   }
 
+  @Test // DRILL-4733
+  public void testMultilevelParquetWithSchemaChange() throws Exception {
+    try {
+      test("alter session set `planner.enable_decimal_data_type` = true");
+      testBuilder()
+          .sqlQuery(String.format("select max(dir0) as max_dir from dfs_test.`%s/src/test/resources/multilevel/parquetWithSchemaChange`",
+              TestTools.getWorkingPath()))
+          .unOrdered()
+          .baselineColumns("max_dir")
+          .baselineValues("voter50.parquet")
```

Comment:
@jinfengni 
I guess confusion in here is that `voter50.parquet` is folder name. If would be clearer I can rename folder and files in it (currently files have names 0_0_0.parquet).